components/common/src/main/java/com/hotels/styx/common/Stopwatch.java

```diff
@@ -0,0 +1,262 @@
+/*
+  Copyright (C) 2013-2020 Expedia Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
```

Comment:
The original copyright header says:

```
/*
 * Copyright (C) 2008 The Guava Authors
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 * http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
```

I'm not sure if we should retain that instead of an Expedia copyright header. Perhaps better to seek clarification from @massdosage ?

Reply:
If this class now contains a mix of code you've taken from Guava and code you've written yourself then what we normally do is change the copyright line to be `Copyright (C) 2008-2020 Expedian, Inc. and The Guava Authors`. We normally then also add a commented line at the top of the class/method/block with a link to the original source code that we used.

Reply:
Thank you - I'll make changes along those lines.